### ISSUE TITLE
-activate doesn't launch application

### ISSUE THREAD

**Issue by aristidesfl** [julienXX/terminal-notifier#196]
When notification is clicked, nothing happens.

```bash
terminal-notifier -message 'hello' -activate 'com.apple.Safari'
```

macOS 10.11.6
terminal-notifier 1.6.3

**Comment by julienXX** [comment]
@aristidesfl could you try to update to the latest terminal-notifier?

**Comment by aristidesfl** [comment]
Any idea why there is a version installed in rbenv?

```
$ brew upgrade terminal-notifier
Error: terminal-notifier 1.7.1 already installed

$ terminal-notifier --version
terminal-notifier (1.6.3) is a command-line tool to send OS X User Notifications.

$ which terminal-notifier
/Users/user/.rbenv/shims/terminal-notifier
```

**Comment by aristidesfl** [comment]
It seems to be creating that shim automatically out of nowere when executed or something.

**Comment by aristidesfl** [comment]
1.7.1 has the same problem.

`-sender` and `-activate` work unreliably. 

It worked once with Safari, most of the times does nothing, and sometimes activates Transmission.app instead.

**Comment by aristidesfl** [comment]
Also now I'm getting Transmission.app icon instead of iTerm icon:

![screen shot 2017-02-01 at 22 58 15](https://cloud.githubusercontent.com/assets/208149/22528063/ee1f52a2-e8d1-11e6-9682-62a43b535a8d.png)

**Comment by brijazz** [comment]
Same issue here, using 1.7.1. `-activate` works once and then stops functioning.  Need to log out/in to restore functionality.

**Comment by julienXX** [comment]
Fixed in 1.8.0.

**Comment by Voleking** [comment]
Same issue here, using latest terminal-notifier(1.8.0), install by homebrew.
